index.html

```diff
@@ -15,21 +15,39 @@
             document.addEventListener('DOMContentLoaded', () => {
                 var queryString = window.location.search;
                 var urlParams = new URLSearchParams(queryString);
+                // the engineVersion should either be changed here or in the URL (PR number or main)
+                var engineVersion = urlParams.get('engine') ?? 'main';
+                var engineCommitSha = urlParams.get('engineCommitSha');
+                // the engineCommitSha can be changed here or in the URL
+                engineCommitSha =
+                    engineCommitSha && engineCommitSha !== ''
+                        ? engineCommitSha
+                        : '857211e434199c0428234f085c06c6d26bb2d098';
```

Comment:
I don't think that we need specify default value for `hash`

Reply:
done 8d1357c6152e99f6d17c62b11db0273881173bdb